ivy_tests/test_ivy/test_misc/test_dynamic_backend.py

```diff
@@ -0,0 +1,151 @@
+import ivy
```

Comment:
Unless there are some issues with the globals, I think we could have these tests inside `test_backend_handler.py` instead

Reply:
Yeah no worries. I have moved the tests in the `test_backend_handler.py`